### DISCUSSION TITLE
How to best get updates via RSS?

### DISCUSSION THREAD

**Discussion by samGbos** [johnlindquist/kit#611] [Q&A] [answered]
It seems like https://www.scriptkit.com/blog is the best way to get product updates, but I haven't been able to find an RSS link for that page. Is there a link that I just can't find, or could a feed be added for the blog?

**Comment by johnlindquist** [comment, marked as answer]
The blog is actually driven from https://github.com/johnlindquist/kit/discussions/categories/announcements

I pull all the announcement discussions with graphql when building the site. I'll see what I can do to add RSS
